cdap-app-templates/cdap-etl/cdap-etl-lib/pom.xml

```diff
@@ -215,6 +215,28 @@
       <artifactId>uadetector-resources</artifactId>
       <version>2014.04</version>
     </dependency>
+    <dependency>
+      <groupId>org.elasticsearch</groupId>
+      <artifactId>elasticsearch-hadoop-mr</artifactId>
+      <version>${es-hadoop.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>it.unimi.dsi</groupId>
+      <artifactId>fastutil</artifactId>
+      <version>${fastutil.version}</version>
+    </dependency>
+    <!-- elasticsearch-hadoop-mr includes a class, StringUtils, which requires commons-httpclient-->
```

Comment:
StringUtils uses HttpClient? Not blaming you, but you learn how bad design means.